{"name": "Ошибка 400 Bad Request при запросе на принятие заказа без номера заказа.", "status": "failed", "statusDetails": {"message": "AssertionError: Ожидалось: 400 Bad Request, получено 404 {\"code\":404,\"message\":\"Not Found.\"}\nassert 404 == 400\n +  where 404 = <Response [404]>.status_code", "trace": "self = <test_accept_order.TestAcceptOrder object at 0x0000022179DC5480>, courier_id = 471139\n\n    @allure.title('Ошибка 400 Bad Request при запросе на принятие заказа без номера заказа.')\n    @allure.description('Создаем курьера фикстурой courier_id. Отправляется PUT запрос на ручку /api/v1/orders/accept/OrderID?courierId=CourierID без номера заказа. '\n                        ' Проверяем, что код ответа 400 Bad Request и текст ответа \"Недостаточно данных для поиска\"'\n                        ' Курьер удаляется из системы фикстурой courier_id.')\n    def test_accept_order_valid_courier_no_order_id_shows_error_400(self, courier_id):\n        path = f'{config.ACCEPT_ORDER_PATH}?courierId={courier_id}'\n        response = Request.put_no_payload(path)\n        expected_response_message = \"Недостаточно данных для поиска\"\n>       assert response.status_code == 400, f'Ожидалось: 400 Bad Request, получено {response.status_code} {response.text}'\nE       AssertionError: Ожидалось: 400 Bad Request, получено 404 {\"code\":404,\"message\":\"Not Found.\"}\nE       assert 404 == 400\nE        +  where 404 = <Response [404]>.status_code\n\ntests\\test_accept_order.py:38: AssertionError"}, "description": "Создаем курьера фикстурой courier_id. Отправляется PUT запрос на ручку /api/v1/orders/accept/OrderID?courierId=CourierID без номера заказа.  Проверяем, что код ответа 400 Bad Request и текст ответа \"Недостаточно данных для поиска\" Курьер удаляется из системы фикстурой courier_id.", "steps": [{"name": "PUT request без payload", "status": "passed", "parameters": [{"name": "path", "value": "'/api/v1/orders/accept/?courierId=471139'"}], "start": 1739134605111, "stop": 1739134605137}], "start": 1739134605111, "stop": 1739134605138, "uuid": "1a84c07e-849b-45a3-a3ae-1622944b5545", "historyId": "61015486349bf265ea0a032b99834f56", "testCaseId": "61015486349bf265ea0a032b99834f56", "fullName": "tests.test_accept_order.TestAcceptOrder#test_accept_order_valid_courier_no_order_id_shows_error_400", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_accept_order"}, {"name": "subSuite", "value": "TestAcceptOrder"}, {"name": "host", "value": "k-julls"}, {"name": "thread", "value": "10016-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_accept_order"}]}